mmdet/models/losses/ghm_loss.py

```diff
@@ -0,0 +1,140 @@
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+
+from ..registry import LOSSES
+
+
+def _expand_binary_labels(labels, label_weights, label_channels):
+    bin_labels = labels.new_full((labels.size(0), label_channels), 0)
+    inds = torch.nonzero(labels >= 1).squeeze()
+    if inds.numel() > 0:
+        bin_labels[inds, labels[inds] - 1] = 1
+    bin_label_weights = label_weights.view(-1, 1).expand(
+        label_weights.size(0), label_channels)
+    return bin_labels, bin_label_weights
+
+
+@LOSSES.register_module
+class GHMC(nn.Module):
+    def __init__(
+            self,
+            bins=10,
+            momentum=0,
+            use_sigmoid=True,
```

Comment:
If there is no plan to support softmax, this argument can be removed.

Reply:
https://github.com/libuyu/mmdetection/blob/143a1ec5ff42eecf6576252ae91dd5ddeb8de70e/mmdet/models/anchor_heads/anchor_head.py#L59
if "us_sigmoid" removed, this line will be False and an error occurs.